tests/c/var_cleanup_attr.c

```diff
@@ -0,0 +1,23 @@
+//===--- libbuiltins.c - test input file for iwyu -------------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+// The cleanup attribute runs a function when the variable goes out of scope. This
+// test makes sure that we track that function.
+
+#include "var_cleanup_attr-d1.h"
```

Comment:
It would be better to put `freep` into an indirectly included header (`...-i1.h`) so that the test would show explicit diagnostics.